templates/agent/client/requestsSchedule.tsx

```diff
@@ -0,0 +1,13 @@
+import { atom, BoxModel } from 'tldraw'
```

Comment:
Throughout the template, we're being a bit inconsistent about how we use the word "schedule" and "review" etc. Make it more consistent.